internal/scylla/client.go

```diff
@@ -143,18 +145,21 @@ func (c *Client) callAPI(ctx context.Context, method, path string, reqBody, resT
 
 	var buf bytes.Buffer
 
-	fmt.Fprintf(&buf, "[DEBUG] (%d) %s %s:\n\n", resp.StatusCode, req.Method, req.URL)
-	defer func() {
-		fmt.Printf("%s\n\n", &buf)
-	}()
-
 	var body io.Reader = io.TeeReader(io.LimitReader(resp.Body, maxResponseBodyLength), &buf)
 
 	if p, ok := resType.(*[]byte); ok {
 		if *p, err = io.ReadAll(body); err != nil {
+			tflog.Trace(ctx, "failed to read body: "+err.Error(), map[string]interface{}{
+				"code":   resp.StatusCode,
+				"status": resp.Status,
+				"error":  err.Error(),
+			})
 			return fmt.Errorf("error reading body: %w", err)
 		}
-
+		tflog.Trace(ctx, "api call succeeded", map[string]interface{}{
+			"code":   resp.StatusCode,
```

Comment:
It is is still important to log the body. You can expect existing providers like gcp or aws, they all print the whole request/response in trace mode. We need the same

Reply:
fixed